docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -68,14 +68,17 @@ In terms of initiation, Grafana supports:
 
 By default, SP-initiated requests are enabled. For instructions on how to enable IdP-initiated logins, see [IdP-initiated Single Sign-On (SSO)]({{< relref "#idp-initiated-single-sign-on-sso" >}}).
 
-{{% admonition type="warning" %}}
-It is possible to setup Grafana with SAML authentication using Azure AD. However, Azure AD limits the number of groups that can be sent in the SAML assertion to 150. If you have more than 150 groups, Azure AD provides a link to retrieve the groups that only works for OIDC/OAuth workflows. At the moment it is not possible to use this link with SAML authentication in Grafana.
+{{% admonition type="note" %}}
+It is possible to set up Grafana with SAML authentication using Azure AD. However, Azure AD limits the number of groups that can be sent in the SAML assertion to 150. If you have users who belong to more than 150 groups, Azure AD provides a Graph API link to retrieve the groups, which only works for OIDC/OAuth workflows.
```

Comment:
Do we need the following? `If you have users who belong to more than 150 groups, Azure AD provides a Graph API link to retrieve the groups, which only works for OIDC/OAuth workflows.`

It's a bit misleading/contradicting with line 74.

Can we somehow rephrase and merge the two sentences?

Reply:
Reworded as requested